utilities/file_reader.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+
+def read_file(log_file):
```

Comment:
@IssaIan you can change the function name to `read_log_file`.

Reply:
Done @rajeman